runner/src/server/plugins/engine/models/submission/WebhookModel.ts

```diff
@@ -7,8 +7,13 @@ function answerFromDetailItem(item) {
     case "list":
       return item.rawValue;
     case "date":
-      return format(new Date(item.rawValue), "yyyy-MM-dd");
+      let date =
```

Comment:
why have you done this?

Reply:
If a form is submitted with and incomplete date, date-fns will try new Date the undefined value which causes a crash. This prevents the crash on incomplete forms (save per page)

Reply:
I'm not sure this is a good idea. I'm fine to merge it but I'd say you should ping Jen a message about it and get her perspective